Kernel/Devices/USB/UHCIController.cpp

```diff
@@ -128,7 +129,7 @@ void UHCIController::reset()
     write_frnum(0);                                                // Set the initial frame number
 
     // Enable all interrupt types
-    write_frnum(UHCI_USBINTR_TIMEOUT_CRC_ENABLE | UHCI_USBINTR_RESUME_INTR_ENABLE | UHCI_USBINTR_IOC_ENABLE | UHCI_USBINTR_SHORT_PACKET_INTR_ENABLE);
+    write_usbintr(0);
```

Comment:
Interrupts seem to lock us up so I've disabled them for now (we don't really need to worry about `IOC` and friends for the moment anway.